README.md

```diff
@@ -1,70 +1,25 @@
-# Getting Started with Create React App
+# Technical Test
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<img src="src/img/Screenshot.png" alt="Screenshot of Technical Test interface" title="Technical Test" width="400px"><br>
 
-## Available Scripts
+I completed this mock technical test as a project for Command Shift bootcamp. The brief was to create a React app that allows users to enter keywords and returns the relevant images.
 
-In the project directory, you can run:
+Built with: React.<br>
```

Comment:
You dont need to use br tags here. Have a look at [markdown editor](https://jbt.github.io/markdown-editor/) so you can play with what the file will look like without having to do multiple commits to the repo to see the results, have a look through this [guide to markdown](https://www.markdownguide.org/cheat-sheet/) too